addon/components/bread-crumbs.js

```diff
@@ -26,11 +28,29 @@ export default Component.extend({
   tagName: 'ol',
   linkable: true,
   reverse: false,
+  hasSchema: false,
   classNameBindings: ['breadCrumbClass'],
+  attributeBindings: [],
+  itemscope: true,
+  itemtype: 'http://schema.org/BreadcrumbList',
   hasBlock: bool('template').readOnly(),
   currentUrl: readOnly('applicationRoute.router.url'),
   currentRouteName: readOnly('applicationRoute.controller.currentRouteName'),
 
+  init() {
+    this._super(...arguments);
+
+    LinkComponent.reopen({
```

Comment:
We need to reopen the `LinkComponent` to add `attributeBindings` but need to do this behind the `hasSchema` flag. However that is too late in the code execution path. 

Do you have thoughts about? 
We could move the configuration to the `ENV` and reopen the `LinkComponent` in the initializer